litmus/VULKAN/ssw7.litmus

```diff
@@ -0,0 +1,16 @@
+VULKAN ssw7
+"Pass: write-after-read hazard of private memory accesses with SSW.SSW in between"
+{
+P0:r0=0;
+P1:r1=0;
+x=0;
+y=0;
+}
+{
+ssw 0 1;
+ssw 1 2;
+}
+ P0@sg 0,wg 0, qf 0               | P1@sg 1,wg 0, qf 0               | P2@sg 2,wg 0, qf 0               ;
+ ld.priv.sc0 r0, x                | ld.priv.sc0 r1, y                | st.priv.sc0 x, 1                 ;
+exists
+(P1:r1 == 1)
```

Comment:
read value should be 0 -> https://github.com/KhronosGroup/Vulkan-MemoryModel/blob/66c31d9cfcc03b1f5231779fbf3c42dc983b62e2/alloy/tests/ssw7.test#L12

Reply:
updated